src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -163,6 +171,7 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
         };
     }
 
+    // tslint:disable-next-line:cyclomatic-complexity
```

Comment:
Would prefer if you fixed the cyclomatic-complexity rather than disable. Looks like there's too many duplicate checks of `debugConfiguration` and `debugConfiguration.dockerRun`

The VS Code api says `debugConfiguration` will always be defined, so you can trust that and remove all instances of `debugConfiguration &&`

Also you can remove all instances of `debugConfiguration.dockerRun && ` by adding this line to the top of the method:
```
debugConfiguration.dockerRun = debugConfiguration.dockerRun || {};
```

As an aside, we have an open work item to turn on `strictNullChecks` which will enforce `undefined` and `null` typing. You can turn it on in `tsconfig.json` if you want to wade through a bunch of errors to temporarily check just your code.

Reply:
Most all of the values turn into simple assignments, so I'm gonna condense them down into the return below.